src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1617,10 +1618,19 @@ private static void sendMessagesToNonlocalDC(Message<? extends IMutation> messag
             target = targets.get(0);
         }
 
+        Tracing.trace("Sending mutation to remote replica {}", target);
         MessagingService.instance().sendWriteWithCallback(message, target, handler);
         logger.trace("Sending message to {}@{}", message.id(), target);
     }
 
+    private static Replica pickReplica(EndpointsForToken targets)
```

Comment:
I originally created a method so I could use BB to rewrite the result... now that I had stable tests I no longer have that BB logic... so I could move this back...